AuthService/jwtproc/jwtHandler.go

```diff
@@ -0,0 +1,62 @@
+package jwtproc
+
+import (
+	"authservice/models"
+	"authservice/parsistence"
+	"authservice/parsistence/entities"
+	"errors"
+	"time"
+)
+
+type JwtHandler struct {
+	persist parsistence.JwtPersistence
```

Comment:
nit: `persist` is a verb, which is bad for naming variables/properties, it's better to use `persister` here